src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const KEYS = Object.values(robot);
+  const VALUES = Object.keys(robot);
+  const INVERSED_ROBOT = {};
+  const UNIQUE_KEYS = [ ...new Set(KEYS) ];
+
+  if (UNIQUE_KEYS.length !== VALUES.length) {
```

Comment:
The check for unique values is correct, but it's not efficient. You are creating a new Set and then converting it back to an array just to check if there are duplicates. Instead, you could use the 'has' method of the Set object to check if a value is already present before adding it.